cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -105,6 +105,8 @@ spec:
               mountPath: /etc/ceph-csi-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+            - name: host-run-mount
+              mountPath: /run/mount
```

Comment:
having these changes is not enough, you need to do more things
* update cephcsi v2.0.0 to v2.0.1
* update upgrade documentations
* test upgrade is working and all mount points are getting _netdev mount option.

Reply:
Made changes to PR to address the above